src/Core/Components/Nav/FluentNav.razor

```diff
@@ -0,0 +1,36 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@inherits FluentComponentBase
+
+<CascadingValue Value="this" IsFixed="true">
+	<div class="fluent-nav-container@(_navOpen ? "" : " collapsed")">
+		<div id="@Id"
+			role="navigation"
+			class="@ClassValue"
+			style="@StyleValue"
+			@attributes="@AdditionalAttributes">
+			@if (UseHeader)
+			{
+				<header role="none" class="header">
+				<FluentButton Appearance="@ButtonAppearance.Transparent"
+							  IconStart="@ToggleIcon"
+							  Title="@Localizer[Localization.LanguageResource.Nav_ToggleTitle]"
+							  IconOnly="true"
+							  OnClick="@ToggleNavAsync"/>
+				</header>
+			}
+			<div class="body">
```

Comment:
`body` is too common a name. There is too much risk that a class already has this name in the dev project => `nav-body`?